src/devices/Dhtxx/README.md

```diff
@@ -26,6 +26,8 @@ using (Dht11 dht = new Dht11(26))
 }
 ```
 
+Note:  1-wire will not work with Win 10 IoT-Core as the OS cannot switch the pin fast enough between output and input (or viz).
```

Comment:
Small NIT: can we put it in bold and can we also add a link here to the issue where we discussed it and also to the links that have the workaround using 2-pins?

Reply:
... idealy with a schematic